templates/repairs_mechanic.html

```diff
@@ -1 +1,63 @@
-{% include 'layout.html' %}
\ No newline at end of file
+{% include 'layout.html' %}
+
+<div class="corner-title">Umówione naprawy</div>
+
+<!-- add a new part -->
+<div style="background: #f39f32; margin-bottom: 3em; margin-top: 1em;">
+    <div style="text-align: center;"><strong>Dodaj nowy termin</strong></div>
+    <form action="" method="post" style="display: inline-block; margin-left: 35%;">
+        <select name="customer_id" id="customer_id">
+            <option value="" disabled selected>Wybierz klienta</option>
+            {% for customer in customers %}
+            <option id="customer_id" value="{{ customer.id }}">{{ customer.first_name }} {{ customer.last_name }}
+                {% endfor %}
+            </option>
+        </select>
+        <textarea id="car_name" name="car_name" rows="1" cols="30" placeholder="Nazwa Samochodu"></textarea>
+        <label for="start_of_repair">Start Naprawy:</label>
+        <input type="date" id="start_of_repair" name="start_of_repair">
+        <label for="end_of_repair">Koniec Naprawy:</label>
+        <input type="date" id="end_of_repair" name="end_of_repair">
+        <input style="display: inline-block; padding-top: 1.5em;" type="submit" value="Dodaj termin">
+    </form>
+</div>
+
+<div class="wrapper" style="background: gainsboro; padding-left: 7em;">
+    <div style="display: inline-block; width: 29.5%; min-width: 200px;">Nazwa samochodu</div>
+    <div style="display: inline-block; width: 10%; min-width: 100px;">Data startu</div>
+    <div style="display: inline-block; width: 10%; min-width: 100px;">Data końca</div>
+    <div style="display: inline-block; width: 10%; min-width: 100px;">Kwota</div>
+</div>
+
+{% if repairs %}
+<div class="scrollable-mechanic-list">
+    <ul>
+        {% for repair in repairs %}
+        <li class="car_part_box " style="list-style: none; padding: 1em;">
+            <div class="wrapper" style="background: #ffc078; padding-left: 3em;">
+                <div style="display: inline-block; width: 30%; min-width: 200px;">{{ repair.car_name }}</div>
+                <div style="display: inline-block; width: 10%; min-width: 100px;">{{ repair.start_date }}</div>
+                <div style="display: inline-block; width: 10%; min-width: 100px;">{{ repair.end_date }}</div>
+                <div style="display: inline-block; width: 10%; min-width: 100px;">{{ repair.money }}</div>
+            </div>
+        </li>
+        {% endfor %}
+    </ul>
+</div>
+{% endif %}
+
+{% if not repairs %}
+<div>
+    {% if msg %}
+    {% if msg == 'Dodano nowy termin' %}
+    <div class="alert alert-success" role="alert">
+        {{msg}}
+    </div>
+    {% else %}
+    <div class="alert alert-danger" role="alert">
+        {{msg}}
+    </div>
+    {% endif %}
+    {% endif %}
+</div>
+{% endif %}
```

Comment:
Can't scroll to see what is behind footer.
![Zrzut ekranu 2024-03-18 181818](https://github.com/PatrykDu/popcorn-works-fastapi-fullstack-app/assets/113666877/289f9fcd-9014-44c1-94b9-d279afd35ac3)